libs/agora/styles/src/lib/_mixins.scss

```
@@ -53,13 +52,12 @@
       @content;
     }
   } @else {
-    @error 'No value found for `#{$breakpoint}`. '
-            + 'Please make sure it is defined in `$breakpoints` map.';
+    @error 'No value found for `#{$breakpoint}`. Please make sure it is defined in `$breakpoints` map.';
   }
 }
 
 @mixin link-hover {
-  color: var(--color-link);
+  // color: var(--color-link);
```

We think that this should be kept: 

```suggestion
  color: var(--color-link);
```

---

this is usually overridden but I will leave it as a default